data/data-pipeline/data_pipeline/etl/base.py

```diff
@@ -289,6 +307,22 @@ def validate(self) -> None:
                         f"`{geo_field}`."
                     )
 
+        # Check whether data contains expected states
+        states_in_output_df = list(
```

Comment:
nit -- try: `
self.output_df[self.GEOID_TRACT_FIELD_NAME].str[0:2].unique().to_list()``
This should be a string and not need to be coerced, and it's more in-line with pandas to use `to_list()`vs explicitly casting it.

Reply:
cool, i didn't know that about the `to_list` vs casting!